Test/SilacTest.cs

```diff
@@ -310,6 +309,301 @@ public static void TestSilacWhenProteinIsMissing()
             string outputFolder = Path.Combine(TestContext.CurrentContext.TestDirectory, @"TestSilac");
             Directory.CreateDirectory(outputFolder);
             var theStringResult = task.RunTask(outputFolder, new List<DbForTask> { new DbForTask(xmlName, false) }, new List<string> { mzmlName }, "taskId1").ToString();
+            //No assertions, just checking it didn't crash
```

Comment:
i guess this is okay. but it kinda inflates code coverage. I'm not sure this is really what we are after.